mantidproject/vesuvio
# Cleaned system tests

**Description of work:**
The previous system tests had some minor issues that caused them to:
    - Take a long time to run
    - Fail most of the tests performed

I corrected each of the three test scripts so that unnecessary runs are avoided and all tests pass.
Checked results of each test script with past stored results and confirmed they are the expected ones.
In total, 25 system tests are performed automatically now on GitHub Workflows.


**To test:**
Look at latest automated run on GitHub workflows and check the for mvesuvio system tests, 25 tests are run.




Fixes #6 .
